3_Print_formatting.ipynb

```diff
@@ -177,7 +177,31 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## 3.4 Special characters  \n",
+    "## 3.4 What's the value of my variable?\n",
+    "Sometimes you will want to print the name of a variable alongside its value, for example during debugging.\n",
+    "Intuitively you might do this like so:\n",
+    "```python\n",
+    "myvar = pi * 2.4**2\n",
+    "print(f\"mvar={myvar}\")\n",
+    "```\n",
+    "However, Python's f-strings give you a conventient syntax for printing a variable name and it's value:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "myvar = 3.14 * 2.4**2\n",
+    "print(f\"{myvar=}\")"
```

Comment:
I guess this should be: 
```print(f"myvar={myvar}")```

I see the point of this extension (fits nicely in here), but not necessarily agree on the 'convenience' tbh. If for debugging purposes, why not simply:
```print("myvar=",myvar)```. 
Because f-strings are faster (not really applicable for debugging?) and provide a more readable string formatting. 
Could we add the following underneath it?:
```
# Alternatively, myvar can be printed similarly as we've seen in the previous chapter. 
print("myvar=",myvar)
```

Reply:
Well it's an extremely minor point. And I'd accept a PR containing what you're suggesting. But it does technically violate DRY (don't repeat yourself). What if you change the variable name?

With `f"{myvar=}"` you only have to fix it in one place.

Reply:
The idea is to have the students run that snippet and see the effect

Reply:
Well actually the idea is that the students see that `f"{myvar=}"` exists. Not that `f\"myvar={myvar}\"` is wrong.

Reply:
OK I've thought about this a little more:
I could use
```python
print("myvar=", myvar)
```
as the initial example instead of
```python
print(f"myvar={myvar}")
```
if you like? Maybe it's a little clearer?

Reply:
Removed the section. It's not really useful anymore